src/main/webapp/user-page.html

```diff
@@ -16,29 +16,67 @@
 
 <!DOCTYPE html>
 <html>
+  <head>
+    <title>User Page</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="/css/user-page.css">
+    <script src="/js/user-page-loader.js"></script>
+  </head>
 
-<head>
-  <title>User Page</title>
-  <meta charset="UTF-8">
-  <link rel="stylesheet" href="/css/main.css">
-  <link rel="stylesheet" href="/css/user-page.css">
-  <script src="/js/user-page-loader.js"></script>
-</head>
-
-<body onload="buildUI();">
-  <nav>
-    <ul id="navigation">
-      <li><a href="/">Home</a></li>
-    </ul>
-  </nav>
-  <h1 id="page-title">User Page</h1>
-
-  <form id="message-form" action="/messages" method="POST" class="hidden">
-    Enter a new message:
-    <br/>
-    <textarea name="text" id="message-input"></textarea>
-    <br/>
-    <input type="submit" value="Submit">
+  <body onload="buildUI();">
+    <nav>
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
+      </ul>
+    </nav>
+    <h1 id="page-title">User Page</h1>
+   
+    <jsp:directive.taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+    <jsp.directive.taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn"%>
+<div class="userForm">
+  <h2>
+    Input what tea you drank today
```

Comment:
Minor wording suggestion: I think it feels a little more natural and less demanding to phrase it as a question, like "What tea did you drink today?"